pkg/controller/garbagecollector/dump.go

```diff
@@ -122,6 +122,7 @@ namespace=%v
 	}
 }
 
+// NewGonumVertex creates a new gonumVertex.
```

Comment:
I'm sorry to say but these comments do not add any information. Is this common in the "fix golint errors" series of PR?

Reply:
I referred the other comments.
e.g.
https://github.com/kubernetes/kubernetes/blob/de5ec3a2ed44c08d46542ec94bb98d8e117342ee/pkg/controller/daemon/daemon_controller.go#L132-L133
https://github.com/kubernetes/kubernetes/blob/de5ec3a2ed44c08d46542ec94bb98d8e117342ee/pkg/controller/deployment/deployment_controller.go#L100-L101